### DISCUSSION TITLE
How to use Dynatrace with SDK 50/51?

### DISCUSSION THREAD

**Discussion by Phips0812** [expo/expo#30818] [Archived: Questions]
I had Dynatrace properly integrated with SDK 49. I did the upgrade to SDK 51 now and get the following error on my development client build (The preview and production channels seem to work fine.):

```
Unable to resolve "@dynatrace/react-native-plugin/jsx-dev-runtime" from "App.tsx"
```

I think the reason is that babel decides based on the environment if it uses `jsx-runtime` or `jsx-dev-runtime`. It seems that `@dynatrace/react-native-plugin` doesn't ships with `jsx-dev-runtime`.

Anyone got this working?

Versions I use:
```
"expo": "^51.0.0",
"@dynatrace/react-native-plugin": "^2.295.1",
"@babel/core": "^7.25.2",
"@babel/parser": "^7.25.3",
"@babel/preset-env": "^7.25.3",
"babel-preset-expo": "~11.0.0",
```

**Comment by Erozionn** [comment]
`@dynatrace/react-native-plugin` version ^2.297.2 includes the `jsx-dev-runtime`. We saw the same issue before updating to the latest version of Dynatrace.
